shared/src/business/utilities/getFormattedTrialSessionDetails.ts

```diff
@@ -37,12 +43,24 @@ export const setPretrialMemorandumFiler = ({ caseItem }) => {
   return filingPartiesCode;
 };
 
+// type the return
```

Comment:
nit: looks like you've done this. 

I could see creating a type and applying it to the entire function (rather than the params and the return within the function definition itself). Perhaps that's a paradigm we should consider in order to help type ApplicationContext. 🤔